lux/vislib/altair/SymbolMap.py

```diff
@@ -0,0 +1,164 @@
+#  Copyright 2019-2020 The Lux Authors.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from lux.vislib.altair.AltairChart import AltairChart
+import altair as alt
+import us
+import pycountry
```

Comment:
We should list us and pycountry into the requirement.txt. 
I installed these via pip and noticed that pycountry in particular is quite a big installation (10MB). Perhaps we could just use a dictionary for doing this lookup to keep the package minimal?

Reply:
Sounds good, I've swapped `pycountry` out with [iso3166](https://pypi.org/project/iso3166/#description), which enables me to do the same thing with only 36 kb (much much lighter)